Makefile

```diff
@@ -1,5 +1,5 @@
 BUILDDIR := $(PWD)
-CHECKDIRS := src tests utils examples setup.py
+CHECKDIRS := src tests examples setup.py
 DOCDIR := docs
 
 BUILD_ARGS :=  # set nightly to build nightly release
```

Comment:
I think we also need to update the "build" section:

python3 setup.py sdist bdist_wheel $(BUILD_ARGS)

where the build command now takes BUILD_TYPE env var?

Reply:
OK, I've updated the Makefile to set the variable to a default and allow it to be overridden when calling make. For example,

```
BUILD_TYPE=nightly make build
```

You could add some separate make targets if you want to do that, too.

Reply:
Cool, thanks!